framework/platform/bpl/linux/platform_db

```diff
@@ -0,0 +1,2 @@
+management_mode=Multi-AP-Controller-and-Agent
+operating_mode=Gateway
```

Comment:
add new line at the end of file

Reply:
also configure vscode to do that automatically - https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files